plugins/core/set_admin.go

```diff
@@ -37,8 +37,8 @@ func (core *CorePlugin) addAdmin(c cmd.Context) (string, slack.PostMessageParame
 		IsAdmin: true,
 	}
 	if err := core.Bot.DAL.SetMemberIsAdmin(&member); err != nil {
-		log.WithError(err).Error("Failed to make user " + username + " admin")
-		return "Failed to make user admin", noParams
+		log.WithError(err).Error("Failed to update %s's admin status", username)
```

Comment:
Are you actually toggling the `is_admin` field here? It seems like it'll always be set to `true`.

Reply:
The function called:

```go
// SetMemberIsAdmin updates the whether the given member is an admin in the DB
// or returns an error.
func (dal *DAL) SetMemberIsAdmin(member *model.Member) error {
	_, err := dal.db.Model(member).
		Set("is_admin = ?is_admin").
		Update()

	return err
}
```

It loooooooks like setting `is_admin = !is_admin`, could be wrong though? Documentation indicates as much too